src/main/java/com/zoo/animals/Mammal.java

```diff
@@ -1,4 +1,18 @@
 package com.zoo.animals;
 
-public class Mammal {
-}
+public class Mammal extends Animal{
+
+    private String furColor;
```

Comment:
How will I set and get the furColor if there's no getter and setter methods and the property itself is private?

I cannot set it when calling constructor either - so there's no way